doc/sphinx-guides/source/_static/util/default.config

```diff
@@ -1,16 +1,23 @@
-HOST_DNS_ADDRESS	localhost
-GLASSFISH_DIRECTORY	/usr/local/glassfish4
-ADMIN_EMAIL	 
-MAIL_SERVER	mail.hmdc.harvard.edu
-POSTGRES_ADMIN_PASSWORD	secret
-POSTGRES_SERVER	127.0.0.1
-POSTGRES_PORT	5432
-POSTGRES_DATABASE	dvndb
-POSTGRES_USER	dvnapp
-POSTGRES_PASSWORD	secret
-SOLR_LOCATION	localhost:8983
-TWORAVENS_LOCATION	NOT INSTALLED
-RSERVE_HOST	localhost
-RSERVE_PORT	6311
-RSERVE_USER	rserve
-RSERVE_PASSWORD	rserve
+[glassfish]
+HOST_DNS_ADDRESS = localhost
+GLASSFISH_DIRECTORY = /usr/local/glassfish4
+
+[database]
+POSTGRES_SERVER = 127.0.0.1
+POSTGRES_PORT = 5432
+POSTGRES_DATABASE = dvndb
+POSTGRES_USER = dvnapp
+POSTGRES_PASSWORD = secret
+POSTGRES_ADMIN_PASSWORD = secret
+
+[system]
+ADMIN_EMAIL =
+MAIL_SERVER = mail.hmdc.harvard.edu
+SOLR_LOCATION = localhost:8983
+TWORAVENS_LOCATION = NOT INSTALLED
+
+[rserve]
+RSERVE_HOST = localhost
+RSERVE_PORT = 6311
+RSERVE_USER = rserve
+RSERVE_PASSWORD = rserve
```

Comment:
Why is default.config changing? Tabs are being replaced by equals signs? Does this still work with the following scripts?

scripts/installer/install
scripts/installer/install.py
scripts/vagrant/install-dataverse.sh
conf/docker-aio/testdata/scripts/vagrant/install-dataverse.sh

I would prefer this change happen in a separate pull request (and have an issue about what problem we're trying to solve, please). That way, this pull request can be docs only, with no need to test code.

Reply:
@j-n-c and I (and @donsizemore) discussed this at https://matrix.to/#/!AmypvmJtUjBesRrnLM:matrix.org/$uZBP0zdxxTqm1ml2c_gGy-LNsej-K-dkMAlXTYUJ-J8 and decided to create a new issue at #8481 to think about what to do with the out of date default.config file.

I just fully reverted the change to default.config like this:

```
git checkout 89d72e8f7 doc/sphinx-guides/source/_static/util/default.config
git commit
git push
```

I'll mark this conversation resolved. Thanks, all.